src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -2984,7 +2984,8 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                     ASR::storage_typeType local_m_storage = ASR::storage_typeType::Default;
                     llvm_el_types.push_back(get_arg_type_from_ttype_t(asr_tuple->m_type[i], m_abi,
                                             arg_m_abi, local_m_storage, arg_m_value_attr, local_n_dims,
-                                            local_a_kind, is_local_array_type, ASRUtils::intent_local));
+                                            local_a_kind, is_local_array_type,
+                                            ASRUtils::intent_local)->getPointerElementType());
```

Comment:
You can't use `getPointerElementType` (not allowed from LLVM 15 onwards). So please figure out a different way to achieve the same goal that you are trying to by using this API.